ipa-core/src/report/hybrid.rs

```diff
@@ -675,6 +676,40 @@ where
 /// Converted report where shares of match key are replaced with OPRF value
 pub type PrfHybridReport<BK, V> = IndistinguishableHybridReport<BK, V, u64>;
 
+/// After grouping `IndistinguishableHybridReport`s by the OPRF of thier `match_key`,
+/// that OPRF value is no longer required.
+pub type AggregateableHybridReport<BK, V> = IndistinguishableHybridReport<BK, V, ()>;
+
+impl<BK, V> AggregateableHybridReport<BK, V>
```

Comment:
@akoshelev suggested this elsewhere. I'm not opposed to making it it's own struct (especially since we will also want to use it with a larger value type for adding up lots of reports.)

Reply:
maybe I misunderstand Andy's point here, I think he is advocating for writing `impl IndistinguishableHybridReport<BK, V, ()>` rather than not using type aliases at all

Reply:
ahh ok I misunderstood. I have no problem with changing this to what you suggested, @akoshelev

Reply:
turned out we don't need this anymore at all. the ZERO was used to fill out an array, but we use the enum now.